Makefile

```diff
@@ -1,10 +1,21 @@
+local-unit:
+	pytest test/unit
+	
+local-integ:
+	pytest test/integration/blackbox_test.py test/integration/dryrun_mixin_docs_test.py
+	# NOT WORKING: pytest test/integration/integration_test.py
```

Comment:
Does `test_integration.py` actually pass for anyone?

Reply:
@jasonpaulos @michaeldiamant ^^^

Reply:
@tzaffi It doesn't pass for me as of https://github.com/algorand/py-algorand-sdk/pull/298/commits/a47355bf7576e73b06cd8c4f3052f1aec31962b7.  

I spent only a few seconds reviewing.  My guess is the error predates the PR.  My unsubstantiated intuition is that the test is incompatible with running algod in a container.  I'm noting the local disk access attempts to resolve kmd.

Obviously, it's an undesirable state.  However, if we agree the issue predates the PR, then I think we ought to file an issue and circle back later.

```
~/dev/py-algorand-sdk teal-blackbox !2 ─────────────────────────────────────────────────────────────────────────────────────────────────── 12s  py-algorand-sdk 11:55:43 AM
❯ make local-integ
# pytest test/integration/blackbox_test.py test/integration/dryrun_mixin_docs_test.py
pytest test/integration/integration_test.py
============================================================================ test session starts ============================================================================
platform darwin -- Python 3.10.2, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
rootdir: /Users/michael/dev/py-algorand-sdk
collected 0 items / 1 error

================================================================================== ERRORS ===================================================================================
___________________________________________________________ ERROR collecting test/integration/integration_test.py ___________________________________________________________
test/integration/integration_test.py:6: in <module>
    from examples import tokens
examples/tokens.py:27: in <module>
    for directory in listdir(data_dir_path):
E   FileNotFoundError: [Errno 2] No such file or directory: '/Users/michael/node/network/Node/'
```

Reply:
Yes, as far as I can tell this test hasn't been ran ever on CI, so it's likely to be broken in a few ways. I don't think this PR should attempt to fix it, but perhaps putting a comment at the top of it might make sense

Reply:
I'll open an issue on the topic that we can link inline.  I'm in-between a few items, but I'll supply an issue _soon_ and then we can safely resolve the thread.

Reply:
@tzaffi I opened https://github.com/algorand/py-algorand-sdk/pull/307 to bring the thread to a close.  Let's resolve once #307 is merged.

Reply:
Am trying to revive the integration test, but am skipping 2 failures